sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -188,4 +188,14 @@
     <node name="arm_group_position_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
           args="ra_trajectory_controller la_trajectory_controller"/>
   </group>
+
+  <arg name="record_hand_data" value="$(optenv SR_RECORD_HAND_DATA disabled)"/>
+  <arg name="log_directory" value="$(optenv HOME)/shadow_bag_storage/sr_record_all_hand_data"/>
+  <arg name="log_split_size" default="64"/>
+  <arg name="log_topics" value='-e ".*(diagnostics|jog_arm|joint|rh|lh|sh|tf).*" /mechanism_statistics /ros_ethercat/motors_halted /sr_watchdog'/>
```

Comment:
are we sure we want to record sr_watchdog on nuc side? I thought we are avoiding that to reduce traffic

Reply:
Oh yes, that's a really good point, we should figure out what we want that string to be